### ISSUE TITLE
NOTES - ES referral enhancement: ACUT reminder for DWP casees

### ISSUE THREAD

**Issue by IlseFerris** [MN-Script-Team/DHS-MAXIS-Scripts#2924]
Hennepin County worker requested the following enhancement: 
"Can the following line be added to this script for a reminder to the [worker] for DWP, that they need to also contact utility companies and update the ACUT panel?" 

Proposed enhancement requesting feedback (1 of 2 options):
1. Add closing message for cases that are identified as DWP to remind the users to contact the utility company or companies and to add/update the ACUT panel. 
OR 
2. Add fields and functionality to notes and update the ACUT panels for DWP cases. 

Thoughts?

**Comment by Farleight** [comment]
I'm slightly confused by this one. With DWP, we should already have all the required information before we ever send that referral... or has that changed?

**Comment by IlseFerris** [comment]
According to my DWP contact,  its not required to have the client provide shelter/utility that’s optional for deductions, but it's a good reminder to workers that they should do it if they haven't already. That said, I think option 1 will be fine for this one.

**Comment by Farleight** [comment]
That's true I suppose... I still hold to the statement though. DWP is almost entirely based on HH costs, and if a client chooses not to verify the deduction, we shouldn't be calling about it at all... whereas if they choose to verify the deduction, we then _would_ need to get it before we approve anything, wouldn't we?

**Comment by Farleight** [comment]
Also, deduction isn't the right word... it's actually a credit.

**Comment by IlseFerris** [comment]
I'm conferring with one of our DWP supervisors. I will comment when I hear back from her. Thanks for all your knowledge and feedback @Farleight !

**Comment by C-Love** [comment]
This functionality is probably more relevant with the CAF script. It should be considered when rewriting the CAF script.
